pymongo/mongo_client.py

```diff
@@ -201,6 +201,10 @@ def __init__(
             and decoding of custom types.
 
           | **Other optional parameters can be passed as keyword arguments:**
+          - Starting in version 4.0 you can pass a custom SRV service name by
+            using the ``srvServiceName`` option like so::
+
+                MongoClient("mongodb+srv://test22.test.build.10gen.cc/?srvServiceName=customname")
```

Comment:
Let's move this to the bottom of this list (under unicode_decode_error_handler). Also please match the doc formatting and style of the other options here.

Reply:
Done.